src/sphinx_book_theme/assets/styles/components/_search.scss

```diff
@@ -1,8 +1,14 @@
 .bd-search-container {
   margin: 2em;
+  h1:first-child {
```

Comment:
```suggestion
  // Remove the search field on the search page since we have one in the sidebar
  h1:first-child {
```

Reply:
could you group this with the other `display: none` below and put the comment above so it's easy for others to remember why this is being hidden?